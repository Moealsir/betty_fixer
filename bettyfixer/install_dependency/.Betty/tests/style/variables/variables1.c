#include <stdio.h>

int test;

/**
 * main - Entry point
 *
 * Return: Always 0 (Success)
 */
int main(void)
{
int i;
	int j;
	int arr = {1, 2, 3};
	int k;

	putchar('\n');
	return (0);
}
