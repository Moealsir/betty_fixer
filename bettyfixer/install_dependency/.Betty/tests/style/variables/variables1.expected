tests/style/variables/variables1.c:3: ERROR: global variables are not allowed
total: 1 errors, 0 warnings, 19 lines checked
